### ISSUE TITLE
The combination of NPSSMDNMultistreamParametricModel  and BiLSTMResF0NonAttentiveDecoder with use_mdn=True results in training failure

### ISSUE THREAD

**Issue by taroushirani** [nnsvs/nnsvs#171]
Hello,

I tried to use the combination of NPSSMDNMultistreamParametricModel and BiLSTMResF0NonAttentiveDecoder whose use_mdn option is enabled, but I got the error message as follows:

````
Error executing job with overrides: ['model=acoustic_nnsvs_world_multi_ar_mgcf0bap_mdn', 'train=myconfig', 'data=myconfig', 'data.train_no_dev.in_dir=dump/oniku_kurumi/norm/train_no_dev/in_acoustic/', 'data.train_no_dev.out_dir=dump/oniku_kurumi/norm/train_no_dev/out_acoustic/', 'data.dev.in_dir=dump/oniku_kurumi/norm/dev/in_acoustic/', 'data.dev.out_dir=dump/oniku_kurumi/norm/dev/out_acoustic/', 'data.in_scaler_path=dump/oniku_kurumi/norm/in_acoustic_scaler.joblib', 'data.out_scaler_path=dump/oniku_kurumi/norm/out_acoustic_scaler.joblib', '++data.sample_rate=48000', 'train.out_dir=exp/oniku_kurumi/acoustic_nnsvs_world_multi_ar_mgcf0bap_mdn', 'train.log_dir=tensorboard/oniku_kurumi_acoustic_nnsvs_world_multi_ar_mgcf0bap_mdn', 'train.resume.checkpoint=', 'train.pretrained_vocoder_checkpoint=']
Traceback (most recent call last):
  File "/content/nnsvs/recipes/oniku_kurumi_utagoe_db/v4mdn-48k-world/../../..//nnsvs/bin/train_acoustic.py", line 525, in my_app
    vocoder_config=vocoder_config,
  File "/content/nnsvs/recipes/oniku_kurumi_utagoe_db/v4mdn-48k-world/../../..//nnsvs/bin/train_acoustic.py", line 376, in train_loop
    pitch_reg_weight=pitch_reg_weight,
  File "/content/nnsvs/recipes/oniku_kurumi_utagoe_db/v4mdn-48k-world/../../..//nnsvs/bin/train_acoustic.py", line 70, in train_step
    outs = model(in_feats, lengths, out_feats)
  File "/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py", line 1130, in _call_impl
    return forward_call(*input, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/nnsvs/acoustic_models/multistream.py", line 519, in forward
    vuv_inp = torch.cat([x, lf0, y_bap_inp], dim=-1)
TypeError: expected Tensor as element 1 in argument 0, but got tuple
````

I found that the "forward"  function of MDNResF0NonAttentiveDecoder returns the tupple of MDN parameters(pi, sigmas, mu), which causes the torch.cat error at the line 519 in the "forward" function of NPSSMDNMultistreamParametricModel.

**Comment by r9y9** [comment]
Did you use `use_mdn: true` for `BiLSTMResF0NonAttentiveDecoder`? If so, it will raise an error as you mentioned, but it is supposed to do so. In my previous experiments, I found that non-MDN models worked better than MDN ones for F0 prediction models. Therefore, I decided to allow the `NPSSMDNMultistreamParametricModel` class to support MDN models only for mgc and bap streams.  Sorry that it's undocumented.
